react-app/src/components/ExtractionError.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { LinkContainer } from 'react-router-bootstrap';
+import { Button } from 'react-bootstrap';
+import LogList from './LogList';
+
+function ExtractionError(props) {
+  return (
+    <div>
+      <h3 className="page-subtitle">An error occurred during extraction.</h3>
+      <p className="page-text">
+        For more information, view the logs below and/or run extraction again with "Log output debugging information"
+        checked.
+      </p>
+      <div className="button-container">
+        <LinkContainer to="/extract">
+          <Button className="nav-button" size="lg" variant="outline-secondary">
+            Extract New
+          </Button>
+        </LinkContainer>
+      </div>
+      <LogList loggedMessages={props.loggedMessages} />
```

Comment:
Similar to my comment on the `LogList`, but I think it might be nice of the heading and directions you have here stayed at the top of the page and just the list of log messages scrolls. It might be possible to make that change within the `LogList` component, so maybe it would just come for free if you made the change in the `logger-display` branch and updated this branch. If so, then this should be set! Otherwise, it might be nice to add a similar change to this page as well.

Reply:
I tried making the subtitle sticky. However, I ran into the same issue as on the logger-display branch--the LogList item covers it up when you scroll.

Reply:
It sounds like you're doing some exploration for this on the `logger-display` branch. Let's see if we can get it working as expected over on that branch, and then we can evaluate what we'll need to update here!

Reply:
I created a CSS class called sticky-item in the logger-display branch. Once that branch is merged and this one is rebased, I can apply that styling to the subtitle of the error page as well.

Reply:
Since this branch is based off the `logger-display` branch, I actually think  you could rebase this branch against the updated version, and you should just get the one or two new commits from that branch on this one. If that rebase doesn't go smoothly though, we can definitely just hold off until the `logger-display` branch is merged like you said!

Reply:
I think it would be good to carry over the scrolling in the LogList component from the `logger-display` branch to here. When I have the app window small and a few different errors listed, the title and directions scroll up on the page when I scroll down to look at the rest of the errors. I think it would be nice to have the same interaction that the LogFile view has.